pyspecProcScripts/correlation_alignment.py

```diff
@@ -44,16 +46,16 @@ def correl_align(
         This data is not modified.
     tol:            float
                     Sets the tolerance limit for the alignment procedure.
-    indirect_dim:   str
-                    Name of the indirect dimension along which you seek to
-                    align
-                    the transients.
+    repeat_dims:   list
+                    List of names of the dimensions along which you seek to
+                    align.
+    nonrepeat_dims: list
+                    These are indirect dimensions that we are not OK to align
+                    (e.g. the indirect dimension of a 2D COSY experiment).
```

Comment:
this is fine -- add `list` --> `list (default [])`

Reply:
checked